subprojects.csv

```diff
@@ -0,0 +1,91 @@
+kubernetes,api,api-machinery
```

Comment:
same with this

Reply:
derp don't know how this got checked in, thanks